starter-code/index.html

```diff
@@ -9,6 +10,83 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
-<body>
+
+<body class=body>
+  <header class=header>
+    <div>
+      <div class=titlediv>
+        <h1 class=title>
+          Iron Hack
+        </h1>
+      </div>
+      <div class=titlediv>
+        <h1 class=title>
+          Shop
+        </h1>
+      </div>
+    </div>
+  </header>
+
+  <!--products-->
+
+  <div class=products>
+
+    <!--first product-->
+
+    <div class=product1>
+
+      <div class=productname1>
+
+        <span class=name1>
+          IronBobble-Head
+        </span>
+
+      </div>
+
+      <div class=productprice1>
+
+        <span class=price1>
+          $25.00
+        </span>
+
+      </div>
+
+      <div class=product1QTY>
+
+        <form action="/action_page.php">
+          QTY
+          <input type="number" name="QTY">
```

Comment:
Little tip, if you don't want to allow negative values, do:
```
<input id="qty-input" type="number" min="0" name="quantity" value="0" />
```
Keep in mind this is not a perfect solution.